### PR TITLE
Shows "switch users" in dropdown for all global admin users

### PR DESCRIPTION
**Pull request by amprokop**
image

Connects https://github.com/department-of-veterans-affairs/caseflow/issues/3865  

# Testing
- Stubbed out `dependencies_faked` and set it to false locally, verified that the "Switch Users" menu item shows up, and that only the "Log In As User" dialog and not the other items appear on that page
- Verified that the `require_global_admin` check runs when attempting to log in as a user

Not directly tested on UAT or preprod, but the logic here is pretty simple and these aren't userfacing pages, so the risk here is low.

### CODE REVIEW

File: app/views/test/users/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <% content_for :page_title do %>&nbsp &gt &nbspTest Users<% end %>
 
 <%= react_component("TestUsers", props: {
-	currentUser: current_user.to_hash,
-	isGlobalAdmin: current_user.global_admin?,
-	testUsersList: @test_users,
-	appSelectList: Test::UsersController::APPS,
-	epTypes: @ep_types
+  currentUser: current_user.to_hash,
```
Comment (amprokop) [inline app/views/test/users/index.html.erb]: converted tabs to spaces in this file

File: client/app/test/TestUsers.jsx [hunk 2]
```diff
@@ -93,52 +93,53 @@ export default class TestUsers extends React.PureComponent {
...
+            loadingText="Switching users" />
+          <br /><br />
+        <p>
+          Not all applications are available to every user. Additionally,
+          some users have access to different parts of the same application.</p>
+        <strong>App Selector:</strong>
+        <TabWindow
+          tabs={tabs} />
+      </div> }
+      { this.props.isGlobalAdmin &&
```
Comment (amprokop) [inline client/app/test/TestUsers.jsx]: I reordered this page a bit such that the "Log in as user" part appears last, just for better organization.

File: config/routes.rb [hunk 3]
```diff
@@ -133,9 +133,10 @@
     if ApplicationController.dependencies_faked?
       resources :users, only: [:index]
       post "/set_user/:id", to: "users#set_user", as: "set_user"
-      post "/log_in_as_user", to: "users#log_in_as_user", as: "log_in_as_user"
       post "/set_end_products", to: "users#set_end_products", as: 'set_end_products'
     end
+
```
Comment (amprokop) [inline config/routes.rb]: this route is protected by `before_action :require_global_admin, only: :log_in_as_user` in the controller

**Comment by nemodjuric** [comment]
screen shot 2018-01-08 at 10 58 37 am

**Comment by amprokop** [comment]
Hmm I think it's not a big deal because the user wouldn't be able to actually do anything on that page, right?

**Pull request merged**